app/views/welcome/index.html.erb

```diff
@@ -45,5 +45,19 @@
       <% end %>
       </ul>
     <% end %>
+    <% if params[:tab] == "stars" %>
+      <ul>
+        <% @starred_repos.each do |repository| %>
+        <li class = "starred_repo">
+          <%= link_to "#{repository.creator}/ #{repository.name}", repository.url, { class: 'repo_link'} %><br>
+          <span class="repo_description"><%= repository.description %><br></span>
+          <span class="repo_language"><%= repository.language %>&nbsp;&nbsp;</span>
+          <%= link_to repository.stars, repository.stars_url, { class: 'repo_stars'} %>&nbsp;&nbsp;
+          <% if repository.forks > 0 %><%= link_to repository.forks, "/#{repository.name}/forks?creator=#{repository.creator}", { class: 'repo_forks'} %>&nbsp;&nbsp;<% end %>
```

Comment:
@jmejia @icorson3, I could not get this link_to repository.forks to pass additional params via the path helper... I tried forks_show_path(repository.name, creator: repository.creator), but only the first param would pass through. Any thoughts as to why that would be?

Reply:
Hmm, interesting. Looks like you were doing it roughly correctly, specifically with `creator: repository.creator`. The only thing that looks a little off is typically for show routes you'd do something like `fork_path(repo.name)` rather than `forks_show_path`.

Reply:
With this code: ` forks_show_path(repository.name, creator: repository.creator)` I only see one parameter being sent. Can you show how you were sending the second parameter?